src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/DefaultCSharpCodeActionProvider.cs

```diff
@@ -29,6 +33,12 @@ internal class DefaultCSharpCodeActionProvider : CSharpCodeActionProvider
             RazorPredefinedCodeFixProviderNames.RemoveUnusedVariable,
         };
 
+        // This is aspirational :)
+        internal static readonly HashSet<string> SupportedImplicitExpressionCodeActionNames = new HashSet<string>()
+        {
+            // RazorPredefinedCodeFixProviderNames.RemoveUnusedVariable,        // No reason we shouldn't be able to remove a variable defined in an implicit expression
```

Comment:
The comment says there's no reason we shouldn't be able to have this line un-commented. That makes me giggle.